packages/flutter/test/widget/bottom_sheet_test.dart

```diff
@@ -71,6 +73,7 @@ void main() {
     expect(showBottomSheetThenCalled, isFalse);
     expect(find.text('BottomSheet'), findsNothing);
 
+    // ignore: unawaited_futures
```

Comment:
This one does not need to be changed. It is already asserting that the returned future completes as expected.